extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationService.js

```diff
@@ -144,10 +144,33 @@ oppia.factory('ItemSelectionInputValidationService', [
               });
             });
           });
-          areAllChoicesCovered = handledAnswers.every(function(handledAnswer) {
-            return handledAnswer;
-          });
         }
+        var answerChoiceToIndex = {};
+
+        answerGroups.forEach(function(answerGroup, answerIndex) {
+          var rules = answerGroup.rules;
+          rules.forEach(function(rule, ruleIndex) {
+            var ruleInputs = rule.inputs.x;
+            ruleInputs.forEach(function(ruleInput) {
+              var choiceIndex = answerChoiceToIndex[ruleInput];
```

Comment:
Isn't answerChoiceToIndex empty? It seems to be declared on line 148 as an empty dict and then never populated. So wouldn't this end up being undefined?

Reply:
I removed this code here by mistake, I will added it back and checked, (I will probably remove that case now, (https://github.com/oppia/oppia/pull/6350#discussion_r264527449). 
 ```javascript
149         seenChoices.forEach(function(seenChoice, choiceIndex) {
150           answerChoiceToIndex[seenChoice] = choiceIndex;
151         });
```